src/Raven.Server/ServerWide/Commands/Sharding/UpdateBucketRangesCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Sharding;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands.Sharding
+{
+    public class UpdateBucketRangesCommand : UpdateDatabaseCommand
```

Comment:
Is this wired any where? for debugging we do need the ability forcibly to override the sharding config.. but I think it should override the entire ShardingConfiguration class.
So let's remove it for now.

Reply:
reverted